_topic_map.yml

```diff
@@ -108,6 +108,8 @@ Topics:
     Distros: openshift-enterprise,openshift-origin
   - Name: Configuring your firewall
     File: configuring-firewall
+  - Name: Configuring project creation
```

Comment:
@adellape, can this assembly fit in a different bucket? We're trying to keep as many topics out of this sub-folder as possible.

Reply:
Sure... maybe Authentication... or Applications.

Why wouldn't the firewall topic be in something like Networking, then?

Reply:
FYI @huffmanca 	I've moved this to the Authentication guide: http://file.rdu.redhat.com/~adellape/062019/project_config/authentication/configuring-project-creation.html#configuring-project-creation

Reply:
Because the firewall requirement is completely outside of the networking Operator's scope, and I thought that the area buckets were intended to cover the content that worked with the product/Operator areas.

Reply:
@kalexand-rh @adellape 

Were we? Networking includes both the CNO and the Ingress Operator (controller?), and will contain several upcoming SDNs. Plus NetworkPolicy is unrelated to any Operator and also lives there. There's no 1-1 mapping for the networking content to Operators.